generator/README.md

```diff
@@ -9,7 +9,7 @@ Due to the dynamic dependency on NetSNMP, you must build the generator yourself.
 
 ```
 # Debian-based distributions.
-sudo apt-get install unzip build-essential libsnmp-dev # Debian-based distros
+sudo apt-get install unzip build-essential libsnmp-dev snmp libsnmp-base snmp-mibs-downloader golang git curl # Debian-based distros
```

Comment:
The mibs downloader is not required.

Reply:
Updated it to remove it. Waiting on the build to queue / finish.